analytics/tests/test_counts.py

```diff
@@ -102,6 +106,41 @@ def create_message(self, sender: UserProfile, recipient: Recipient, **kwargs: An
             kwargs[key] = kwargs.get(key, value)
         return Message.objects.create(**kwargs)
 
+    def subscribe(self, user_profile: UserProfile, recipient: Recipient) -> None:
+        Subscription.objects.create(user_profile=user_profile, recipient=recipient)
+
+    # Actual creation and sending of messages.
+    def send_personal_message(self, from_user: UserProfile, to_user: UserProfile, content: str="test content",
```

Comment:
Either we can use this or we can add separate logic for creating UserMessage objects in create_user. Whichever you suggest.

Reply:
For these, we should just inherit from `ZulipTestCase` and use the existing send_message helpers of this form from `test_classes.py`